students/18th/songheechoi/js/main.js

```diff
@@ -0,0 +1,55 @@
+
+const commentInput = document.querySelector(".feed__myComment input");
+
+const postButton = document.querySelector(".comment__btn-post")
+const postButton1 = document.getElementsByClassName("comment__btn-post")
+
+const commentsList = document.querySelector(".feed__commentList");
+const logo = document.querySelector(".logo");
+
+// const likeButton = 
+function test(){
+    // console.log(postButton,postButton1);
+    if(commentInput.value){
+        postButton.style.opacity ="1";
+    }else{
+        postButton.style.opacity ="0.3";
+    }
+}
+commentInput.addEventListener("input",test)
+commentInput.addEventListener("keyup", function(e){
+    // console.log(e);
+    if(e.keyCode ===13&&commentInput.value){
```

Comment:
이렇게 가독성을 높이면 더 좋을거 같아요!
```suggestion
    if(e.keyCode ===13 && commentInput.value){
```